packages/core/integration-tests/test/css-modules.js

```diff
@@ -575,7 +575,7 @@ describe('css modules', () => {
       },
     );
 
-    assert.deepEqual(res, [['page2', 'foo_4fY2uG foo_1ZEqVW foo_j1UkRG']]);
+    assert.deepEqual(res, [['page2', '_4fY2uG_foo _1ZEqVW_foo j1UkRG_foo']]);
```

Comment:
It's odd that (AFAICT) all classes start with an underscore, apart from this last one

Reply:
The underscore is added when the hash starts with a number, otherwise we'd have to escape it.